websocket.go

```diff
@@ -0,0 +1,105 @@
+package bitstamp
+
+import (
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"github.com/gorilla/websocket"
+)
+
+const bitstampWsUrl = "wss://ws.bitstamp.net"
+
+type WsEvent struct {
+	Event   string      `json:"event"`
+	Channel string      `json:"channel"`
+	Data    interface{} `json:"data"`
+}
+
+type WsClient struct {
+	ws       *websocket.Conn
+	done     chan bool
+	sendLock sync.Mutex
+	Stream   chan *WsEvent
+	Errors   chan error
+}
+
+func NewWsClient() (*WsClient, error) {
+	c := WsClient{
+		done:   make(chan bool, 1),
+		Stream: make(chan *WsEvent),
+		Errors: make(chan error),
```

Comment:
let's make this buffered too, and send to it with `select default`

Reply:
I don't understand what did you mean, describe it, plz

Reply:
`Errors: make(chan error, 1)`
+
line 52:
```
select {
   case c.Errors <- err:
   default:
}
```
in case nobody cares about those errors.

Reply:
fixed it